src/test/java/uk/nhs/hee/tis/trainee/forms/api/validation/FormRPartAValidatorTest.java

```diff
@@ -197,4 +200,29 @@ void shouldNotValidateSubmittedForm() {
     assertThat("Unexpected number of errors.", fieldErrors.size(), is(0));
     verifyNoInteractions(formRPartARepositoryMock);
   }
+
+  @ParameterizedTest
+  @NullSource
+  @ValueSource(strings = {""})
```

Comment:
`@NullAndEmptySource` will do this for you

Reply:
Oops 🤦‍♂️